examples/multimodal/speech_llm/conf/modular_audio_gpt_config_eval.yaml

```diff
@@ -0,0 +1,124 @@
+name: megatron_audio_gpt_eval
```

Comment:
add some info on what this config is for.

Reply:
added